Cargo.toml

```diff
@@ -126,11 +126,11 @@ strum = { version = "0.25.0", features = ["derive"] }
 tracy-client = "0.17"
 thiserror = "1"
 wgpu = { version = "0.20.0", path = "./wgpu", default-features = false }
-wgpu-core = { version = "0.20.0", path = "./wgpu-core" }
+wgpu-core = { version = "22.0.0", path = "./wgpu-core" }
 wgpu-example = { version = "0.20.0", path = "./examples/common" }
 wgpu-macros = { version = "0.20.0", path = "./wgpu-macros" }
 wgpu-test = { version = "0.20.0", path = "./tests" }
-wgpu-types = { version = "0.20.0", path = "./wgpu-types" }
+wgpu-types = { version = "22.0.0", path = "./wgpu-types" }
```

Comment:
All of these should be 22

Reply:
Weirdly, the `wgpu-example` entry doesn't exist. I've removed it, and bumped the others.